### DISCUSSION TITLE
Pods class accepting arrays of arrays, or arrays of objects

### DISCUSSION THREAD

**Discussion by pglewis** [pods-framework/pods#6646] [Ideas]
The idea of this would be to allow the Pods class to work off of an array or object, abstracting what we already do in PodsUI, which would need some adjustments in PodsData, but everything wins with this. Example: `$pod = pods( $data );` or `$pod = pods( $data, $row_number );` This could also give us the ability to traverse arrays/objects in the same way we do for relationships, assuming the arrays are multidimensional. This could definitely work for field(), but would be interesting to see how it could translate for find(), though it would not be able to accept SQL syntax, it would need our meta_query field handling.

**Comment by sc0ttkclark** [comment]
This is a prerequisite to #109's 'fields' output type for loop field data.

**Comment by sc0ttkclark** [comment]
Putting on hold until after CMB2 prototyping is done.
